db/migrate/20190430190333_add_price_to_products.rb

```diff
@@ -0,0 +1,5 @@
+class AddPriceToProducts < ActiveRecord::Migration[5.2]
+  def change
+    add_column :products, :price, :integer
+  end
+end
```

Comment:
Do you really need 4 migrations to create this table and its columns?

Reply:
No that's right. We changed so that we only have and added columns directly into the Product DB migration file.